OpenRA.Test/OpenRA.Game/ShapeTest.cs

```diff
@@ -0,0 +1,96 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.IO;
+using NUnit.Framework;
+using OpenRA.Mods.Common.HitShapes;
+
+namespace OpenRA.Test
```

Comment:
This appears to be in the wrong place.  The file path implies that it is a Game test, but it then depends on things from mods.common.  Can you put this in a new Mods.Common dir instead?